server/routes/api/documents/documents.ts

```diff
@@ -1034,6 +1052,9 @@ router.post(
           }
         );
         authorize(user, "createChildDocument", parentDocument, { collection });
+      } else if (document.isWorkspaceTemplate) {
+        const team = await Team.findByPk(document.teamId);
+        authorize(user, "createDocument", team);
```

Comment:
```suggestion
        authorize(user, "createDocument", user.team);
```

Combined with suggested change on L940